development/nautobot_config.py

```diff
@@ -147,6 +147,20 @@
         "hashicorp_vault": {
             "url": os.environ.get("HASHICORP_VAULT_URL"),
             "token": os.environ.get("HASHICORP_VAULT_TOKEN"),
+            # "vaults": {
+            #     "hashicorp_approle": {
+            #         "url": os.environ.get("HASHICORP_VAULT_URL"),
+            #         "auth_method": "approle",
+            #         "role_id": os.getenv("NAUTOBOT_HASHICORP_VAULT_ROLE_ID"),
+            #         "secret_id": os.getenv("NAUTOBOT_HASHICORP_VAULT_SECRET_ID"),
+            #     },
+            #     "hashicorp_v1_custom_mount": {
+            #         "url": os.environ.get("HASHICORP_VAULT_URL"),
+            #         "token": os.environ.get("HASHICORP_VAULT_TOKEN"),
+            #         "kv_version": "v1",
+            #         "default_mount_point": "secret_kv",
+            #     },
+            # }
```

Comment:
I'd prefer to have the dev environment use the most current configuration instead of the backwards compatible legacy configuration. What was the reason for rolling this back?

Reply:
Sounds great to me! That was an accidental commit, and the tests were coded to use the settings. I will add this back in and fix the tests.